rust-runtime/smithy-json/src/deserialize/token.rs

```diff
@@ -30,7 +30,7 @@ impl<'a> EscapedStr<'a> {
 
     /// Unescapes the string and returns it.
     /// If the string doesn't need unescaping, it will be returned directly.
-    pub fn to_unescaped(&self) -> Result<Cow<'a, str>, EscapeError> {
```

Comment:
I think this will break the fuzz tester for smithy-json, or, IIRC, that's the place where I needed this to be `&self` instead of `self`.

Reply:
self is `Copy` so this shouldn't change anything, but I'll check it.